pkg/util/ecs/host_tags.go

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2020-present Datadog, Inc.
+
+//go:build docker
+
+package ecs
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/datadog-agent/comp/core/tagger/tags"
+	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
+	"time"
+)
+
+// GetTags returns host tags or static tags that are automatically added to
+// ECS metrics
+func GetTags(ctx context.Context) ([]string, error) {
+	ctx, cancel := context.WithTimeout(ctx, 4*time.Second)
+	defer cancel()
+
+	cluster, _, err := getECSMetadata(ctx)
```

Comment:
Should we cache this result or can is change within the lifespan of the Agent ?
The other host tags providers cache their result since we will query them every 30 min.